api/graphql/schema.graphql

```diff
@@ -1905,7 +1905,7 @@ input SyncJobPosterTemplateSkillInput {
 }
 
 input UpdateJobPosterTemplateSkillsInput {
-  attach: [AttachJobPosterTemplateSkillInput!]
+  connect: [ConnectJobPosterTemplateSkillInput!]
```

Comment:
I don't mind it, I'm just curious why you made this change?

Reply:
Attach is [not a proper operation](https://lighthouse-php.com/6/eloquent/nested-mutations.html#belongstomany) so no data was being saved for skills with it.